### PR TITLE
Increment duplicates on import

### LINKED ISSUE #1196: importText: Duplicate Thought
I think it is a regression from 354c6f32b.

## Steps to Reproduce

``` 
- a
- a
``` 

Copy and paste the above text.

## Current Behavior

- a
- a

## Expected Behavior

- a

### PR DESCRIPTION
**Pull request by shresthabijay**
fixes #1196

### CODE REVIEW

**Comment by shresthabijay** [comment]
@raineorshine I have made the changes and added tests too. Please review the changes. Thanks!

**Comment by raineorshine** [comment]
@onurpolattimur Could you look into this browserstack failure? It should be getting the username from the repo secret.

```
Run browserstack/github-actions/setup-env@master
Error: Action Failed: Error: Action input failed for reason: Input required and not supplied: username
```

**Review by raineorshine — CHANGES_REQUESTED** [review]
Looks good!

File: src/util/importJSON.ts [hunk 1]
```diff
@@ -94,11 +94,19 @@ const saveThoughts = (state: State, contextIndexUpdates: Index<Parent>, thoughtI
     }
   }
 
-  const updates = blocks.reduce((accum, block, index) => {
+  const updates = blocks.reduce< ThoughtIndices & {
+    duplicateIndex: Index<number>,
+  }>((accum, block, index) => {
```
[the remark this answers is not in the document]
Comment (shresthabijay) [reply src/util/importJSON.ts:99]: Sure.

**Review by raineorshine — APPROVED** [review]

**Pull request merged**